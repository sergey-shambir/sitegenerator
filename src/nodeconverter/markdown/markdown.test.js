const { convertMarkdown } = require('./markdown');
const { join } = require('path');

const testDataDir = join(__dirname, 'test_data');

describe("convertMarkdown function", () => {
    it("can process images", () => {
        const markdownPath = join(testDataDir, 'images.md')
        const html = convertMarkdown(markdownPath);
        expect(html).toMatchSnapshot();
    })
})