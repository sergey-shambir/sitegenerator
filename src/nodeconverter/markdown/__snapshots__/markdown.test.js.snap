// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convertMarkdown function can process images 1`] = `
"<nav class="table-of-contents"><ol><li><a href="#Izobrazheniya">Изображения</a></li></ol></nav><h1 id="Izobrazheniya" tabindex="-1">Изображения</h1>
<figure data-type="image"><img src="img/image.jpg?v=9eac7817" alt="Изображение в формате JPEG" loading="lazy" decoding="async"></figure>
<figure data-type="image"><img src="img/image.png?v=7633de90" alt="Изображение в формате PNG" loading="lazy" decoding="async"><figcaption>Ещё и с заголовком</figcaption></figure>
"
`;
